README.md

```diff
@@ -20,6 +20,16 @@ This project is based on an old version of this Android example: https://github.
 #### Streaming from Ubuntu (or anything running PulseAudio)
 The following web page describes how to configure PulseAudio for use with this player.  http://kaytat.com/blog/?page_id=301
 
+in /etc/pulse/default.pa:
```

Comment:
Capitalize i.  If you want to add this, I suggest adding a little more  context for the user.

Reply:
done.